R/pkg/R/schema.R

```diff
@@ -123,6 +123,7 @@ structField.character <- function(x, type, nullable = TRUE) {
   }
   options <- c("byte",
                "integer",
+               "float",
```

Comment:
Could you have a test for create a DataFrame with `float` type? It may crash now.